pkg/sql/catalog/descs/descriptor.go

```diff
@@ -575,14 +586,4 @@ func isTemporarySchema(name string) bool {
 // validationLevelMap is used to map indexes in a slice to their required
 // validation level. This avoids any heap allocations as compared to a
 // slice containing the correspondence between an index to a validation level.
-type validationLevelMap struct {
-	m util.FastIntMap
-}
-
-func (vlm *validationLevelMap) get(idx int) catalog.ValidationLevel {
-	return catalog.ValidationLevel(vlm.m.GetDefault(idx))
-}
-
-func (vlm *validationLevelMap) set(idx int, l catalog.ValidationLevel) {
-	vlm.m.Set(idx, int(l))
-}
+type validationLevelMap []catalog.ValidationLevel
```

Comment:
nit: looks like we don't need this type alias anymore, also with "Map" in the naming for an slice seems interesting.